packages/api-media-karma/src/index.ts

```diff
@@ -19,10 +19,12 @@ export class MediaServerError extends Error {
 export class KarmaMediaAdapter implements MediaAdapter {
   readonly url: URL
   readonly token: string
+  readonly internalURL?: URL
 
-  constructor(url: URL, token: string) {
+  constructor(url: URL, token: string, internalURL: URL = url) {
     this.url = url
     this.token = token
+    this.internalURL = internalURL
```

Comment:
I suggest you set `this.internalURL` to `internalURL ?? url` and then use `this.internaURL` for `_uploadImage()` and `deleteImage`.

Reply:
`internalURL` already has `url` as its default value so I am not sure that would change anything. In `_uploadImage` I needed the ` ?? this.url`, for typescript reasons, but I don't think that's even necessary because I can't see how it would be undefined.

Reply:
Ahhh now i see. you're using `url` as the default paramter. Okay but then you can remove the `?` from line 22 and then you shouldn't need `??` on line 36.